bin/ci/semver.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function grep_word_from_git_commits_since_last_tag() {
+    local commits=$(git log --oneline $(git describe --tags --abbrev=0 @^)..@ --grep="$1" 2> /dev/null)
+
+    if [ $? -eq 128 ]; then
+        echo 2
+    elif [ -z "$commits" ];
+    then
+        echo 0
+    else
+        echo 1
+    fi
+}
+
+function get_version() {
+    if [ `grep_word_from_git_commits_since_last_tag "BREAKING CHANGE"` -gt 0 ]
+    then
```

Comment:
```suggestion
    if [ "$(grep_word_from_git_commits_since_last_tag "BREAKING CHANGE")" -gt 0 ]; then
```
https://github.com/koalaman/shellcheck/wiki/SC2006
https://github.com/koalaman/shellcheck/wiki/SC2046